app/src/main/kotlin/com/wire/android/workmanager/worker/DeleteConversationLocallyWorker.kt

```diff
@@ -71,6 +79,26 @@ class DeleteConversationLocallyWorker @AssistedInject constructor(
         } ?: Result.failure()
     }
 
+    override suspend fun getForegroundInfo(): ForegroundInfo {
+        notificationChannelsManager.createRegularChannel(
+            NotificationConstants.OTHER_CHANNEL_ID,
+            NotificationConstants.OTHER_CHANNEL_NAME
+        )
+
+        val notification = NotificationCompat.Builder(applicationContext, NotificationConstants.OTHER_CHANNEL_ID)
+            .setSmallIcon(R.drawable.notification_icon_small)
+            .setAutoCancel(true)
+            .setSilent(true)
+            .setCategory(NotificationCompat.CATEGORY_SERVICE)
+            .setProgress(0, 0, true)
```

Comment:
suggestion: I think we don't need to show any progress on this notification

Reply:
god damn copy-past 😅